lib/InputTest.ts

```diff
@@ -749,6 +749,14 @@ export namespace InputTest {
  *
  * ---
  *
+ * Note: Firefox will ignore the option `deviceId` if it is passed to the test
+ * and will instead prompt the user for a device via a Firefox specific GUI.
+ * This is due to the Firefox implemenation of the
+ * `navigator.mediaDevices.getUserMedia` function, which is used to capture
+ * media devices.
```

Comment:
As someone who is not familiar with the behavior, the documentation doesn't really describe to me what I should expect differently on Firefox. I think we need to explain how the browser behaves, and how these 3 relates: system default settings, firefox permission settings, deviceId parameter